tests/e2e/test_e2e.py

```diff
@@ -188,42 +188,42 @@ def test_e2e():
     featurizer.apply(split=1, train=True, parallelism=PARALLEL)
     assert session.query(Feature).count() == 214
     num_feature_keys = session.query(FeatureKey).count()
-    assert num_feature_keys == 1281
+    assert num_feature_keys == 1024
```

Comment:
Why number of feature keys is different?

Reply:
I couldn't find a way to debug this easily, to know which parts are increasing the number of feature keys. Any suggestion?

Reply:
You can dump two sets of features into files and compare the difference.  You can compare the features by feature type (e.g., textual, tabular, structural, visual) and the difference should come from the new changes.